src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -422,141 +393,188 @@ Relativamente all'utilizzo della #glossary("dashboard"), viene definito un unico
+  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary[dashboard] superamento soglie.
+
+#figure(
+  image("assets/UML/UC4.4_Visualizzazione-tabella-superamento-soglia-precipitazioni.png",width:70%),
+  caption: [UC4.4 Visualizzazione tabella superamento soglia di precipitazioni]
+)
+
+=== UC4.5: Visualizzazione tabella superamento soglia di inquinamento dell'aria
+- *Attore Principale*: #glossary("amministratore pubblico")\;
+- *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente una tabella, la quale mostra tutti i dati superanti la soglia di 80#[#sym.mu]g su metro cubo ($80#sym.mu g\/m^3$), mostrando il valore superante la soglia, il #glossary("sensore") che ha rilevato tale valore e il timestamp relativo;
+- *Scenario Principale*:
+  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary[dashboard] superamento soglie.
+
+#figure(
+  image("assets/UML/UC4.5_Visualizzazione-tabella-superamento-soglia-inquinamento.png",width:70%),
+  caption: [UC4.5 Visualizzazione tabella superamento soglia di inquinamento dell'aria]
+)
+
+=== UC4.6: Visualizzazione tabella superamento soglia dei bacini idrici
+- *Attore Principale*: #glossary("amministratore pubblico")\;
+- *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente una tabella, la quale mostra tutti i dati superanti la soglia del 70% di capienza, mostrando il valore superante la soglia, il #glossary("sensore") che ha rilevato tale valore e il timestamp relativo;
+- *Scenario Principale*:
+  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary[dashboard] superamento soglie.
+
+#figure(
+  image("assets/UML/UC4.6_Visualizzazione-tabella-superamento-soglia-bacini.png",width:70%),
+  caption: [UC4.6 Visualizzazione tabella superamento soglia dei bacini idrici]
+)
+
+=== UC4.7: Visualizzazione tabella superamento soglia delle zone ecologiche
+- *Attore Principale*: #glossary("amministratore pubblico")\;
+- *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente una tabella, la quale mostra tutti i dati superanti la soglia del 80% di capienza, mostrando il valore superante la soglia, il #glossary("sensore") che ha rilevato tale valore e il timestamp relativo;
+- *Scenario Principale*:
+  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary[dashboard] superamento soglie.
+
+#figure(
+  image("assets/UML/UC4.7_Visualizzazione-tabella-superamento-soglia-cassonetti.png",width:70%),
+  caption: [UC4.7 Visualizzazione tabella superamento soglia delle zone ecologiche]
+)
+
+=== UC5: Ricezione notifiche di superamento soglie
```

Comment:
Purtroppo penso che questo sia sbagliato.
La notifica è un'interazione che il sistema ha con l'utente, e non l'utente con il sistema.
Anche in alcune parti dell'esame di Cardin, requisiti di questo tipo (ovvero il sistema manda una notifica ad un utente dopo un determinato evento) non venivano modellati nell'UML, appunto perché non sono interazioni dirette dell'utente con il sistema.
@ranghetto Ti ricordi?

Provo a spiegarmi meglio. Dire che la ricezione è l'azione è un problema. Dovrebbe essere più un visualizzazione lista notifiche superamento soglie, che ricezione, poiché questa ricezione non prevede un'azione attiva da parte dell'utente.

@nance97 @Dovid308

Reply:
non ci avevo pensato, rimetto a visualizzazione notifiche di superamento soglie questo UC e le sue specializzazioni dato che di fatto l'utente le visualizza su discord? facciamo UC5 vis. lista, UC 5.1 vis. singola notifica e UC 5.2, UC 5.3, etc.. vis. notifica temperatura, etc.. come specializzazioni di UC 5.1

Reply:
si ne abbiamo parlato stamattina ed effettivamente ha senso